airbyte-server/src/main/java/io/airbyte/server/handlers/HealthCheckHandler.java

```diff
@@ -44,7 +44,7 @@ public HealthCheckHandler(ConfigRepository configRepository) {
   public HealthCheckRead health() {
     boolean databaseHealth = false;
     try {
-      databaseHealth = configRepository.getStandardWorkspace(PersistenceConstants.DEFAULT_WORKSPACE_ID) != null;
+      databaseHealth = configRepository.getStandardWorkspace(PersistenceConstants.DEFAULT_WORKSPACE_ID, false) != null;
```

Comment:
true here too. we don't want the healthcheck to fail because the default workspace was deleted.